_cohorts/ols-1.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Why did you create this file and for other cohorts? Such files already exist

Reply:
I cant find any that has or that i can edit to include what i need
can you point me in the right direction?

Reply:
It doesn't include any of the variables i used. Should i add them?

And you suggested I reuse the code for the table in the card but I don't think that is possible because each cohort was assigned its own variable.

Reply:
Which variables do you need that is not in the Markdown file or in the metadata files in _data folder?

Reply:
the name of the cohort {'OLS-1, OLS-2, etc} and the link to the respective .md file

If you see in the table that's currently in the .md file; each cohorts name and link are written down separately and not given as variables

Reply:
True. With the current structure of the data, it is complicated to loop on the different cohorts. For now, you can remove the loop there, do like in the table for each cohort and create the cards manually

Reply:
ok then! I'll try do that
and include the date at the front of the card

is there anything i should put at the back of the card, or should i leave it as is?